compiler/luci/pass/src/FuseMulWithFullyConnectedPass.cpp

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
plz add `FuseMulWithFullyConnectedPass.test.cpp`

Reply:
in the unit test, you should add `count(positive) <= count(negative)` where negative method name ends with `_NEG`